src/test/java/org/dplava/git/RepositoryCommitValidatorTest.java

```diff
@@ -52,6 +52,15 @@ public void testValidRepository() throws GitAPIException, IOException, Interrupt
 
         // now test the ability to skip dozens of irrelevant files
 
+        //test unique IDs
+        FileUtils.copyFile(new File("src/test/resources/sample-valid.xml"), new File(gitDir, "sample1.xml"));
+        git.add().addFilepattern(".").call();
+        c = git.commit().setAuthor("test", "test@fake.fake")
+                .setMessage("Added files with duplicate IDs")
+                .setCommitter("committer", "committer@fake.fake").call();
+        v.queueForValidation(gitUrl, c.getName(), r);
+        v.waitFor(gitUrl, c.getName());
+        assertEquals("Error: Files \"sample.xml\" and \"sample1.xml\" have the same id.", reports.getFailureReport(gitUrl, c.getName()));
```

Comment:
It's possible the order that files are listed differ between OS's.  In any case I get:

`org.junit.ComparisonFailure: expected:<Error: Files "sample[.xml" and "sample1].xml" have the same ...> but was:<Error: Files "sample[1.xml" and "sample].xml" have the same ...
`
If you add the appropriate static import and change this line to:

`assertTrue("An error about duplicate ids should be reported!", reports.getFailureReport(gitUrl, c.getName()).endsWith(" have the same id."));
`